stable/grafana/templates/deployment.yaml

```diff
@@ -130,6 +130,12 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+        {{- if .Values.command }}
```

Comment:
`command` seems to be missing in values.yaml

Reply:
`command` in value file is optional (this is why it's checked using `if`)

Reply:
we need it only when we want to use HashiCorp Vault (https://banzaicloud.com/blog/inject-secrets-into-pods-vault/)

Reply:
As a user, I want to know about this.
This should be added to values.yaml and documentd in README.

Reply:
files have been updated